src/Domain/Entity/Promotion.php

```diff
@@ -25,7 +25,7 @@ class Promotion
     /** @var int */
     private $weighting;
 
-    /** @var  bool */
+    /** @var bool */
```

Comment:
Single line comments also only have 1 leading start don't they?

Reply:
It's a docblock rather than a general comment so it has [two asterisks](https://docs.phpdoc.org/guides/docblocks.html). If you don't have two asterisks then phpdoc won't pick it the annotations within the comment.